<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QWidget" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="TitleWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1921</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>1861</width>
     <height>881</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="TrainManager" name="Main_Page"/>
   <widget class="QWidget" name="FESCar_Page"/>
   <widget class="QWidget" name="FES_Page">
    <widget class="QPushButton" name="select">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="new_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>新建</string>
     </property>
    </widget>
   </widget>
   <widget class="SettingWidget" name="setting_Page"/>
   <widget class="UserManager" name="userMsg_Page"/>
  </widget>
  <widget class="QPushButton" name="advanced_Btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>高级设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trainPage_Btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>训练界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="jipian_Btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>极片脱落</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spasm_Btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>痉挛报警</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emergency_Btn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>急停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="channel_Btn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>通道</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allParam_Btn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>全参数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_Btn">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TrainManager</class>
   <extends>QWidget</extends>
   <header>trainmanager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitleWidget</class>
   <extends>QWidget</extends>
   <header>titlewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UserManager</class>
   <extends>QWidget</extends>
   <header>usermanager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingWidget</class>
   <extends>QWidget</extends>
   <header>settingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
